geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -2212,15 +2212,20 @@ public Properties getSecurityProperties() {
      * @param oldSystem the old DS, which is in a partially disconnected state and cannot be used
      *        for messaging
      */
-    void reconnecting(InternalDistributedSystem oldSystem);
+    default void reconnecting(InternalDistributedSystem oldSystem) {
```

Comment:
Is this change to default still required? It looks like we are overriding for each implementation of ReconnectListener.

Reply:
It's not required but I think it's a good addition. This is an internal listener, so we're not impacting any users. Adding default just means that implementers (tests) can implement just the method(s) it needs to instead of including empty impl methods.